docs/reference/units/manifest-entry.rst

```diff
@@ -114,27 +116,64 @@ depends on data from the manifest can look like this::
 ``manifest`` job is not related to the ``manifest.ns`` value. Since any
 provider can ship additional manifest entries and then all share the flat
-name-space of resource attributes looking at the ``.ns`` attribute is a way to
+namespace of resource attributes looking at the ``.ns`` attribute is a way to
 uniquely identify a given manifest entry.
 
 Collecting Manifest Data
 ------------------------
 
-To interactively collect manifest data from a user please include this job
-somewhere early in your test plan:
-``com.canonical.plainbox::collect-manifest``.
+When running Checkbox, if some jobs in the selected test plan depend on a
+manifest entry, a System Manifest screen will be presented so that the user
+can define the value for each required manifest entries, for example:
+
+.. code-block:: none
+
+     System Manifest:
+    ┌──────────────────────────────────────────────────────────────────────────────┐
+    │ Does this machine have the following graphics ports?                         │
+    │   DVI                                    ( ) Yes   ( ) No                    │
+    │   DisplayPort                            ( ) Yes   ( ) No                    │
+    │   HDMI                                   ( ) Yes   ( ) No                    │
+    │   VGA                                    ( ) Yes   ( ) No                    │
+    │ Does this machine have this piece of hardware?                               │
+    │   A Wi-Fi Module                         ( ) Yes   ( ) No                    │
+    │   A fingerprint reader                   ( ) Yes   ( ) No                    │
+    │   An Ethernet Port                       ( ) Yes   ( ) No                    │
+    │   Audio capture                          ( ) Yes   ( ) No                    │
+    │   Audio playback                         ( ) Yes   ( ) No                    │
+    │   Thunderbolt 3 Support                  ( ) Yes   ( ) No                    │
+    │   Touchpad                               ( ) Yes   ( ) No                    │
+    │   Touchscreen                            ( ) Yes   ( ) No                    │
+    └──────────────────────────────────────────────────────────────────────────────┘
+     Press (T) to start Testing                                      Shortcuts: y/n
+
+User can quickly fill these by using the ``y`` and ``n`` keyboard shortcuts,
+or highlight an entry and select the right answer using the arrow and the
+``Space`` keys.
 
 Supplying External Manifest
 ---------------------------
 
-The manifest file is stored in
-``$HOME/.local/share/plainbox/machine-manifest.json``.
-If the provisioning method ships a valid manifest file there it can be used for
-fully automatic but manifest-based deployments.
+The manifest file is stored in ``/var/tmp/checkbox-ng/machine-manifest.json``.
+If the provisioning method ships a valid manifest file there it can be used
+for fully automated manifest-based deployments.
+
+Here is an example of such a file:
```

Comment:
Is it an empty file when Checkbox is installed, or is the file created only after the first time we run manifest?

Reply:
It does not exist when Checkbox is installed. It is only created when a test plan that requires some manifest info is launched.